pkg/v1/sdk/features/Dockerfile

```diff
@@ -15,7 +15,11 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 COPY apis/ apis/
 COPY cli/runtime cli/runtime
+<<<<<<< HEAD
```

Comment:
Looks like rebase error here.

Reply:
oops, thanks for catching this, fixed it